libs/features/personalization/personalization.js

```diff
@@ -51,6 +51,10 @@ const DATA_TYPE = {
   TEXT: 'text',
 };
 
+export const MERCH_CARD_COLLECTION_SELECTOR = 'in-card-collection';
+
+const CUSTOM_SELECTORS = [MERCH_CARD_COLLECTION_SELECTOR];
```

Comment:
this too.

Reply:
this is the smallest footprint to have to get "custom" actions defined so mep knows it just needs to register them as such.

Reply:
If selectors had a pattern or prefix, we wouldn't need to add custom selectors in this file.

Reply:
if we did that, you wouldn't need the constant.  You just extract the 2nd part of the string and in your block you just use your own block name.

Reply:
sure :) or custom:whatever but yes in that case in-block:merch-card-collection sounds right indeed :)